### PR TITLE
Update README. Add License.

### PR DESCRIPTION
**Pull request by jalessio**
The README links will make sense once the new directory re-org changes land in the urbanfootprint repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,27 @@
...
+submodule](https://github.com/CalthorpeAnalytics/urbanfootprint/blob/master/.gitmodules) in the main
+[UrbanFootprint project repository](https://github.com/calthorpeanalytics/urbanfootprint). For
+instructions on how to properly configure an UrbanFootprint instance please refer to
+[UrbanFootprint's top-level
+README](https://github.com/CalthorpeAnalytics/urbanfootprint/blob/master/README.md).
+
+## Creating Your Own Client Configurations
+
+The recommended pattern for creating a client configuration for your own organization is to [fork
+this repository](https://help.github.com/articles/fork-a-repo/). You can then substitue the URL for
```
Comment (janet) [inline README.md]: small typo `substitue` > `substitute`

File: README.md [hunk 2]
```diff
@@ -1 +1,27 @@
...
+your repository in the setup instructions. For example, in place cloning the sample `urbanfootprint-
```
Comment (janet) [inline README.md]: typo `in place` > `in place of`

File: README.md [hunk 3]
```diff
@@ -1 +1,27 @@
...
+configuration` repository like this:
```
Comment (janet) [inline README.md]: syntax: there is a space between `urbanfootprint-` and `configuration`

File: README.md [hunk 4]
```diff
@@ -1 +1,27 @@
...
+
+    git clone https://github.com/CalthorpeAnalytics/urbanfootprint.git urbanfootprint git clone
+    https://github.com/CalthorpeAnalytics/urbanfootprint-configuration.git urbanfootprint-
```
Comment (janet) [inline README.md]: syntax: there is a space between urbanfootprint- and configuration

File: README.md [hunk 5]
```diff
@@ -1 +1,27 @@
...
+    configuration
+
+you would clone *your* forked `urbanfootprint-configuration` repository like this:
+
+    git clone https://github.com/CalthorpeAnalytics/urbanfootprint.git urbanfootprint git clone
+    https://github.com/YOUR-ORGANIZATION/urbanfootprint-configuration.git urbanfootprint-
```
Comment (janet) [inline README.md]: syntax: there is a space between urbanfootprint- and configuration

**Comment by jalessio** [comment]
@janet I addressed all your comments

**Pull request merged**